core/modules/user/user.module

```diff
@@ -1411,26 +1411,27 @@ function user_menu() {
     'menu_name' => 'user-menu',
   );
 
+  $user_login_page_tabs = user_login_page_tabs();
   $items['user/login'] = array(
     'title' => 'Log in',
     'page callback' => 'backdrop_get_form',
     'page arguments' => array('user_login'),
     'access callback' => 'user_is_anonymous',
-    'type' => MENU_DEFAULT_LOCAL_TASK,
+    'type' => ($user_login_page_tabs ? MENU_DEFAULT_LOCAL_TASK : MENU_CALLBACK),
```

Comment:
The position of the closing parenthesis is irritating.
Also two times below, when it's used the same way.

Reply:
I don't understand why the closing parenthesis annoys you, but we probably can remove both opening and closing..

Reply:
I really should have been clearer in my comment. :wink: 

What expected is:
```
($user_login_page_tabs) ? MENU_DEFAULT_LOCAL_TASK : MENU_CALLBACK,
```
The result seems OK with both parenthesis positions, but I found it irritating, when wrapped as a whole. (Note: not annoyed, just irritated :wink: ) Maybe that's just me... :shrug:

Reply:
I don't know if we have standard but I don't see examples in core where the whole thing is in parentheses. And parentheses are only used on the first part if it's doing some comparison.